packages/mcdoc/src/runtime/checker/index.ts

```diff
@@ -877,18 +877,14 @@ function simplifyIndexed<T>(
 	const child = simplify(typeDef.child, { ...context, typeArgs: [] })
 
 	if (child.kind !== 'struct') {
-		context.ctx.logger.warn(`Tried to index unindexable type ${child.kind}`)
+		context.ctx.logger.warn(`Tried to index un-indexable type ${child.kind}`)
 		return { kind: 'union', members: [] }
 	}
 	let values: McdocType[] = []
 
 	for (const index of typeDef.parallelIndices) {
 		let lookup: string[] = []
 		if (index.kind === 'static') {
-			if (index.value === '%fallback') {
-				values = child.fields.filter(f => f.kind === 'pair').map(f => f.type)
-				break
```

Comment:
Is this removal intended? Does `foo: minecraft:item[%fallback]` still work?

Reply:
@NeunEinser would know, he said I should remove it

Reply:
(Answered in Discord)